### PR TITLE
Add Question RUD api tests

### PR DESCRIPTION
**Pull request by workshy**
@yuz123 
@VitaliyaRyabchuk 
@YuriiKononenko 
@sydorenkooleksii 
Review please

### CODE REVIEW

File: src/test/java/automation/examples/api/business/questions/AllQuestionsDefinitionSteps.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import static java.lang.String.format;
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AllQuestionsDefinitionSteps {
+
+    @Autowired
+    private AllQuestions allQuestions;
+
+    private QuestionDTO STORED_QUESTION;
```
Comment (YuriiKononenko) [inline src/test/java/automation/examples/api/business/questions/AllQuestionsDefinitionSteps.java]: pls,use java code convention naming.

File: src/test/java/automation/examples/api/common/CommonSteps.java [hunk 2]
```diff
@@ -29,7 +30,8 @@
     private CustomerHelper customerHelper;
 
     public void setApiRequestSpecification() {
-        request = given().contentType(JSON).accept(JSON).baseUri(baseURL);
+        requestSpecification = new RequestSpecBuilder().setContentType(JSON).setAccept(JSON).setBaseUri(baseURL).build();
+//        request = given().contentType(JSON).accept(JSON).baseUri(baseURL);
```
Comment (YuriiKononenko) [inline src/test/java/automation/examples/api/common/CommonSteps.java]: pls,remove comment

**Review by YuriiKononenko — APPROVED** [review]

File: src/test/java/automation/examples/api/business/questions/AllQuestionsDefinitionSteps.java (hunk 1 above)
Comment (workshy) [reply src/test/java/automation/examples/api/business/questions/AllQuestionsDefinitionSteps.java]: Done, thx

**Review by yuriytkachenko-ideals — APPROVED** [review]

File: src/test/java/automation/examples/api/common/CommonSteps.java (hunk 2 above)
Comment (workshy) [reply src/test/java/automation/examples/api/common/CommonSteps.java]: Done, thx

**Review by aqaexplorer — APPROVED** [review]

**Pull request merged**